app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,12 @@
 class PostsController < ApplicationController
   def index
-    @user_index = params[:user_id]
+    @user_path = "/users/#{params[:user_id]}"
```

Comment:
- [ ] <b>OPTIONAL:</b> What do you need `@user_path` Every time you create a route, rails generate a method that will return the path of that route. what you have to do is to call `user_path` if it is user, or `post_path` if is post, etc.

Reply:
yeah, I got some issues with that, that I want to prevent just right now